### ISSUE TITLE
Architecture problem

### ISSUE THREAD

**Issue by Qamamri** [sundowndev/phoneinfoga#1028]
While trying to install phoneinfoga, by this command line
curl -sSL https://raw.githubusercontent.com/sundowndev/phoneinfoga/master/support/scripts/install | bash
it says "Your OS/Arch is not supported."

I'm running Kali Linux on VM on a mac M1 silicon.

**Comment by Qamamri** [comment]
Linux_aarch64

**Comment by sundowndev** [comment]
aarch64 is not yet supported, I'll update this issue accordingly.

**Comment by cyb3rw0lf** [comment]
In the meanwhile the `Linux_arm64` package seems to be working fine:
```
curl -sSL https://raw.githubusercontent.com/sundowndev/phoneinfoga/master/support/scripts/install | sed -e 's/os=.*/os="Linux_arm64"/' | bash
```

**Comment by yogaakash12** [comment]
──(root㉿kali)-[/home/yoga]
└─# bash <( curl -sSL https://raw.githubusercontent.com/sundowndev/phoneinfoga/master/support/scripts/install )
Error: Linux_aarch64 is not supported, installation will attempt to proceed manually.

Please check the releases page for a list of supported systems.
https://github.com/sundowndev/phoneinfoga/releases

Read more at https://sundowndev.github.io/phoneinfoga/install/


/proc/self/fd/11: line 146: jq: command not found
Please select a version of PhoneInfoga to download.
Validating checksum ...
######################################################################################################################################################################## 100.0%
sha256sum: phoneinfoga_checksums.txt: no file was verified
                                                                                                                                                                               
┌──(root㉿kali)-[/home/yoga]
└─# curl -sSL https://raw.githubusercontent.com/sundowndev/phoneinfoga/master/support/scripts/install | sed -e 's/os=.*/os="Linux_arm64"/' | bash        
Error: Linux_aarch64 is not supported, installation will attempt to proceed manually.

Please check the releases page for a list of supported systems.
https://github.com/sundowndev/phoneinfoga/releases

Read more at https://sundowndev.github.io/phoneinfoga/install/


main: line 146: jq: command not found
Please select a version of PhoneInfoga to download.
Validating checksum ...
######################################################################## 100.0%
sha256sum: phoneinfoga_checksums.txt: no file was verified
check this error

**Comment by sundowndev** [comment]
@yogaakash12 You have to install [jq](https://jqlang.github.io/jq/).

**Comment by WhiteDevil5787** [comment]
1st run this
git clone https://github.com/sundowndev/PhoneInfoga
cd PhoneInfoga/

Then
wget $(curl -s https://api.github.com/repos/sundowndev/phoneinfoga/releases/latest | grep tarball_url | cut -d '"' -f 4) -O PhoneInfoga.tar.gz
tar -xvzf PhoneInfoga.tar.gz
cd sundowndev*

Then
# Add --help at the end of the command for a list of install options
bash <( curl -sSL https://raw.githubusercontent.com/sundowndev/phoneinfoga/master/support/scripts/install )



Then boom 💥💥💥💥 it will get installed

**Comment by yerdaulete** [comment]
when I wanted to execute this ./build-alpine   I got unsupported architecture: aarch64
someone knows what happened

**Comment by ShadowComes** [comment]
> 1st run this git clone https://github.com/sundowndev/PhoneInfoga cd PhoneInfoga/
> 
> Then wget $(curl -s https://api.github.com/repos/sundowndev/phoneinfoga/releases/latest | grep tarball_url | cut -d '"' -f 4) -O PhoneInfoga.tar.gz tar -xvzf PhoneInfoga.tar.gz cd sundowndev*
> 
> Then
> 
> # Add --help at the end of the command for a list of install options
> bash <( curl -sSL https://raw.githubusercontent.com/sundowndev/phoneinfoga/master/support/scripts/install )
> 
> Then boom 💥💥💥💥 it will get installed

YES! This worked! Thanks! In this year it was working like in a normal way using docker or compose... But now it's again not working.. But this fixed it!
